### PR TITLE
Add support for parsing traces inputs

### PR DESCRIPTION
**Pull request by medvedev1088**
Note: people are not supposed to include all table definitions for functions in PRs https://github.com/blockchain-etl/ethereum-etl-airflow/issues/49 as only information not covered by logs should be included (rare case). We should mention here https://github.com/blockchain-etl/ethereum-etl-airflow/issues/49 that only table definitions for events should be included (unless calls are needed for specific use case).

### CODE REVIEW

**Comment by askeluv** [comment]
Sorry @medvedev1088 I had some un-merged changes in a separate branch that I've now merged into master. Could you please fix the conflicts and when you push again, you can test to see if everything works by using the links that the `now` bot posts.

File: api/requirements.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ eth-utils==1.8.1
 MarkupSafe==1.1.1
 Jinja2==2.10.3
 sanic==19.6.0
+sanic-cors==0.9.9.post4
```
Comment (askeluv) [inline api/requirements.txt]: I went back to using Flask so we don't need the Sanic libraries

**Comment by medvedev1088** [comment]
@askeluv not sure you need to grant me access to the now application, I get 404 when accessing this url: https://zeit.co/nvk/abi-parser/r3fjx76z4.

**Comment by askeluv** [comment]
Sorry I meant the Preview link: https://abi-parser-git-fork-blockchain-etl-master.nvk.now.sh

**Comment by medvedev1088** [comment]
I've done some smoke testing - seems to be working fine. 

@askeluv I'm thinking if we should only include events when user clicks "Download Table Definitions" and for calls they can download individual files if necessary. And probably rename button to "Download Table Definitions for Events"  The reason is my note in the description:

> Note: people are not supposed to include all table definitions for functions in PRs blockchain-etl/ethereum-etl-airflow#49 as only information not covered by logs should be included (rare case).

wdyt?

**Comment by askeluv** [comment]
Yeah that makes sense to me @medvedev1088. Should we do that in a separate PR or do you want to include it here?

File: frontend/src/components/Downloader.js [hunk 2]
```diff
@@ -24,11 +25,11 @@ export class Downloader extends Component {
                 }
               ),
               {
-                rename: ({url, index, urls}) => ContractName + '_event_' + Object.entries(tables)[index][0] + '.json'
+                rename: ({url, index, urls}) => ContractName + '_event_' + events[index][0] + '.json'
               }
             );
           }}
-        >Download Table Definitions</Button>
+        >Download Table Definitions for Events"</Button>
```
Comment (askeluv) [inline frontend/src/components/Downloader.js]: Remove trailing `"`

**Comment by medvedev1088** [comment]
Just included it here. It's ready to be merged now.

**Comment by askeluv** [comment]
Deployed to https://contract-parser.d5.ai/

**Pull request merged**